cmd/multiclusterapp.go

```diff
@@ -138,17 +162,70 @@ func MultiClusterAppCommand() cli.Command {
+						Name: "role,r",
+						Usage: "Set roles required to launch/manage the apps in target projects. Specified roles on " +
+							"upgrade will override all the original roles. Leave it empty to keep current roles",
+					},
 					cli.BoolFlag{
 						Name:  "show-versions,v",
 						Usage: "Display versions available to upgrade to",
 					},
+				},
+			},
+			cli.Command{
+				Name:        "add-project",
+				Usage:       "Add target projects to a multi-cluster app",
+				Action:      addMcappTargetProject,
+				Description: "Examples:\n #Add 'p1' project in cluster 'mycluster' to target projects of a multi-cluster app named 'myapp'\n rancher multiclusterapp add-project myapp mycluster:p1\n",
+				ArgsUsage:   "[APP_NAME/APP_ID, CLUSTER_NAME:PROJECT_NAME/PROJECT_ID...]",
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "answers,a",
+						Usage: "Path to an answers file that provides overrided answers for the new target projects, the format of the file is a map with key:value. Supports JSON and YAML",
+					},
+					cli.StringFlag{
+						Name:  "values",
+						Usage: "Path to a helm values file that provides overrided answers for the new target projects",
+					},
 					cli.StringSliceFlag{
-						Name: "target,t",
-						Usage: "Target project names/ids to upgrade. Specified targets on upgrade will override all " +
-							"the original targets. Leave it empty to keep current targets",
+						Name:  "set",
+						Usage: "Set overrided answers for the new target projects",
 					},
 				},
 			},
+			cli.Command{
+				Name:        "delete-project",
+				Usage:       "Delete target projects from a multi-cluster app",
+				Action:      deleteMcappTargetProject,
+				Description: "Examples:\n #Delete 'p1' project in cluster 'mycluster' from target projects of a multi-cluster app named 'myapp'\n rancher multiclusterapp delete-project myapp mycluster:p1\n",
+				ArgsUsage:   "[APP_NAME/APP_ID, CLUSTER_NAME:PROJECT_NAME/PROJECT_ID...]",
+			},
+			cli.Command{
+				Name:        "add-member",
+				Usage:       "Add members to a multi-cluster app",
+				Action:      addMcappMember,
+				Description: "Examples:\n #Add 'user1' and 'user2' as the owners of a multi-cluster app named 'myapp'\n rancher multiclusterapp add-member myapp owner user1 user2\n",
+				ArgsUsage:   "[APP_NAME/APP_ID, ACCESS_TYPE, USER_NAME/USER_ID...]",
+			},
+			cli.Command{
+				Name:        "delete-member",
+				Usage:       "Delete members from a multi-cluster app",
+				Action:      deleteMcappMember,
+				Description: "Examples:\n #Delete the membership of a user named 'user1' from a multi-cluster app named 'myapp'\n rancher multiclusterapp delete-member myapp user1\n",
+				ArgsUsage:   "[APP_NAME/APP_ID, USER_NAME/USER_ID...]",
+			},
+			cli.Command{
+				Name:        "list-members",
+				Aliases:     []string{"lm"},
+				Usage:       "List current members of a multi-cluster app",
+				Description: "List current members of a multi-cluster app",
```

Comment:
Either update description to be different or drop it, having them the same makes the help output look funny.